piximi-documentation/technical-faq.md

```diff
@@ -49,22 +51,26 @@ You can also serve the application locally using Docker. The instructions to do
 
 No internet connection is necessary to save or load projects and models.
```

Comment:
@ErinWeisbart can you caveat this? You don't need an internet connection for _classifier_ models but you do for segmentation models (and for Cellpose you need to full on send them to the internet)

Reply:
done. (FYI I had edited the technical FAQ for formatting but not content. Happy to add any additional content edits you suggest)

Reply:
TY! That was the only content issue I noticed but will add others if I see them